packages/frontend/src/pages/_app.tsx

```diff
@@ -3,7 +3,6 @@ import type { ReactElement, ReactNode } from "react";
 import type { NextPage } from "next";
 import type { AppProps } from "next/app";
 import { UserProvider } from "@auth0/nextjs-auth0/client";
-
 import "@/styles/globals.css";
 
 export type NextPageWithLayout<P = object, IP = P> = NextPage<P, IP> & {
```

Comment:
why use a `getLayout` function when u can just wrap the layout?

Reply:
i previously have two different layout for two categories of pages, but eventually i change it to only one, and I'm lazy to change it back